docs/devices/TroubleShooting.rst

```diff
@@ -47,12 +47,12 @@ Alternatively, use assisted GPS (A-GPS) to speed up the time to fix.
 
 .. only:: v1_5_x
 
-    One form of A-GPS is using SUPL and it can be optionally enabled in the firmware using the `SUPL client  <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/supl_os_client.html>`_.
+    One form of A-GPS is using SUPL and it can be optionally enabled in the firmware using the `SUPL client  <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/others/supl_os_client.html>`_.
```

Comment:
```suggestion
    One form of A-GPS is using SUPL and it can be optionally enabled in the firmware using the `SUPL client  <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.5.1/nrf/include/supl_os_client.html>`_.
```

**suggestion:** link to 1.5.1 page from 1.5.x branch

Reply:
Fixed.